_data/events.yml

```diff
@@ -317,3 +317,126 @@
     title: View meetup event
     target: _target
+
+- title: Coding Club with Women Coding Community
+  description: We are excited to announce our next Coding Club session as a part of
+    Women Coding Community!
+  category_style: coding-club
+  category_name: Coding Club
+  date: Thu, Oct 3, 2024
+  expiration: '20241003'
+  host: Nonna Shakhova
+  speaker: Busra Ecem Sakar
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/event-coding-club-3.jpg
+    alt: Coding Club with Women Coding Community  cover photo
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303263034/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: 'The Power of Negotiation: Women Claiming Their Worth in Tech'
+  description: "Join the Women Coding Community for an insightful and empowering session\
+    \ on the Power of Negotiation. In this career talk, you will learn practical strategies\
+    \ to advocate for yourself effectively, ensuring that you can confidently negotiate\
+    \ fair compensation and professional opportunities. Whether you're preparing for\
+    \ your first job or aiming to advance in your career, this session will offer\
+    \ valuable advice on building negotiation skills that lead to success. Don\u2019\
+    t miss this opportunity to invest in your future!"
+  category_style: career-talk
+  category_name: Career Talk
+  date: Tue, Oct 8, 2024
+  expiration: '20241008'
+  host: Madhura Chaganty
+  speaker: Kirsty Harland
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/career-talk-20241008.webp
+    alt: 'The Power of Negotiation: Women Claiming Their Worth in Tech cover photo'
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303139655/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: Writing Club - applying the Pyramid Principle in your writing
+  description: We are excited to continue with our Writing Club sessions as a part
+    of Women Coding Community!
+  category_style: writing-club
+  category_name: Writing Club
+  date: Thu, Oct 10, 2024
+  expiration: '20241010'
+  host: Revolut
+  speaker: Julia Babahina, Co-Founder / Director of Women Coding Community and a Lead
+    Risk Management Specialist at
+  time: 6:00 PM UTC
+  image:
+    path: /assets/images/events/event-writing-club.jpeg
+    alt: Writing Club - applying the Pyramid Principle in your writing  cover photo
+  link:
+    path: https://www.meetup.com/women-coding-community/events/303682675/?eventOrigin=group_events_list
+    title: View meetup event
+    target: _target
+
+- title: Design Patterns Course
+  description: Join Women Coding Community for our next session on design patterns!
+  category_style: tech-talk
+  category_name: Tech Talk
+  date: Wed, Oct 16, 2024
+  expiration: '20241016'
+  host: Revolut
+  speaker: Julia Babahina, Co-Founder / Director of Women Coding Community and a Lead
```

Comment:
I think this should be Irina

Reply:
In the screenshot it says host revolut

Reply:
Good spot! I actually just checked the first two and last to see if the automation worked correctly 🫢

Reply:
Looks like it was because the descriptions for those events were not written to conform to the format we typically use (which is how the `extract_name` function that gets the host/speaker names was configured)

I have updated the descriptions and they now extract correctly.